app/account/managers.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth.base_user import BaseUserManager
+
+
+class UserManager(BaseUserManager):
+    use_in_migrations = True
+
+    def _create_user(self, username=None, email=None, phone=None, password=None, **extra_fields):
+        if not username:
+            if not email or not phone:
```

Comment:
Здесь я имел в виду, что можно условия объединить)

```suggestion
        if not username or (not email and not phone):
```

И в целом ниже условия как-то не оптимально написаны. user создаешь несколько раз. Подумай над условиями, предприми необходимые действия, если чего-то не хватает и в итоге создай юзера